web/src/hashchange.js

```diff
@@ -95,7 +95,6 @@ export function changehash(newhash) {
     if (browser_history.state.changing_hash) {
         return;
     }
-    maybe_hide_recent_topics();
```

Comment:
justify removing

Reply:
hash changes can now move from one recent topics UI to another, so we shouldn't hide

though maybe see if we can remove this before all these changes? show it's not needed